packages/xstate-wallet/src/store/indexedDB-backend.ts

```diff
@@ -61,11 +62,40 @@ export class IndexedDBBackend implements DBBackend {
       request.onerror = err => reject(err);
       request.onsuccess = () => {
         this._db = request.result;
+
         resolve(request.result);
       };
     });
   }
 
+  public async dump(): Promise<object> {
```

Comment:
Is `dump` the right name for this? Maybe `serialize` or `toJSON` or something?

Reply:
Perhaps, but it is not something I put a lot of thought into.

The purpose of this is to log the store contents for debugging purposes. So, `dump` may seem more appropriate than `serialize`, considering its usage in programming.

I'm not fussed either way.